internal/route/api/v1/repo/path.go

```diff
@@ -0,0 +1,200 @@
+package repo
+
```

Comment:
Super weird. I had renamed path.go to contents.go and then started accepting all the UI suggestions and began committing. Looks like all the commits went to path.go file instead of contents.go :(